src/coreclr/gc/gc.cpp

```diff
@@ -49461,13 +49649,14 @@ void gc_heap::do_pre_gc()
 #ifdef TRACE_GC
     size_t total_allocated_since_last_gc = get_total_allocated_since_last_gc();
 #ifdef BACKGROUND_GC
-    dprintf (1, (ThreadStressLog::gcDetailedStartMsg(),
+    //dprintf (1, (ThreadStressLog::gcDetailedStartMsg(),
```

Comment:
Dead code? Can we remove this line?

Reply:
thanks for noticing! unfortunately it did not show there were new messages when I looked at the issue (only after I merged already so I will address this in another PR).